sdk/resourcemanager/azure-resourcemanager/src/test/java/com/azure/resourcemanager/TestTrafficManager.java

```diff
@@ -337,12 +345,14 @@ public void print(TrafficManagerProfile profile) {
                     .append("\n\t\t\tId: ")
                     .append(endpoint.id())
                     .append("\n\t\t\tType: ")
-                    .append(endpoint.endpointType())
-                    .append("\n\t\t\tTarget resourceId: ")
-                    .append(endpoint.targetAzureResourceId())
-                    .append("\n\t\t\tTarget resourceType: ")
-                    .append(endpoint.targetResourceType())
-                    .append("\n\t\t\tMonitor status: ")
+                    .append(endpoint.endpointType());
+                if (!isPlaybackMode) {
+                    info.append("\n\t\t\tTarget resourceId: ")
+                        .append(endpoint.targetAzureResourceId())
+                        .append("\n\t\t\tTarget resourceType: ")
+                        .append(endpoint.targetResourceType());
+                }
```

Comment:
Why this needs the condition? I assume this is only for logging?

Reply:
Fixed in the new version.

Reply:
The `targetResourceRegion` has been replace to `Sanitized` with latest TestProxy.

Reply:
Fixed what?

I mean, if this is just for logging, let it print. Why does it matter?

Reply:
The implementation of `targetResourceType` is as follows:
```
    @Override
    public TargetAzureResourceType targetResourceType() {
        return new TargetAzureResourceType(
            ResourceUtils.resourceProviderFromResourceId(targetAzureResourceId()),
            ResourceUtils.resourceTypeFromResourceId(targetAzureResourceId()));
    }
```
The specified ID `Sanitized` is not a valid Azure resource ID

Reply:
I still didn't get it. Does call of `targetResourceType()` throws when id is not valid?

If it does not throw, invalid output like `null/null` does not matter much, if it is only some output.

Reply:
When the value of `targetResourceId` is replaced with `Sanitized`, an exception is thrown with the following message: `The specified ID Sanitized is not a valid Azure resource ID.`